Link to external sqlite
Code was already prepater, just comment the correct block instead
author: micketeer@gmail.com
--- src/CMakeLists.txt.orig	2026-09-30 20:36:12.110902528 +0200
+++ src/CMakeLists.txt	2026-09-30 20:36:30.587627829 +0200
@@ -40,33 +40,33 @@
 # SQLITE3
 
 # for dynamically linking with system sqlite3
-# find_package(SQLite3 REQUIRED)
-# target_link_libraries(gx SQLite::SQLite3)
+find_package(SQLite3 REQUIRED)
+target_link_libraries(gx PRIVATE SQLite::SQLite3)
 
 # For building and statically linking sqlite3
-include(FetchContent)
-FetchContent_Declare(
-  sqlite3
-  URL      https://www.sqlite.org/2024/sqlite-amalgamation-3460100.zip
-  URL_HASH SHA3_256=af6aae8d3eccc608857c63cf56efbadc70da48b5c719446b353ed88dded1e288
-)
-FetchContent_MakeAvailable(sqlite3)
-add_library(sqlite3 STATIC ${sqlite3_SOURCE_DIR}/sqlite3.c)
-target_compile_options(sqlite3 PRIVATE -w)  # disable warnings
-
-target_compile_definitions(sqlite3 PRIVATE
-    SQLITE_ENABLE_API_ARMOR=1
-    SQLITE_THREADSAFE=2
-    SQLITE_ENABLE_FTS5=1
-    SQLITE_ENABLE_FTS3_PARENTHESIS=1
-    SQLITE_ENABLE_RTREE=1   # may need for loc-indexing
-    SQLITE_ENABLE_COLUMN_METADATA=1
-    SQLITE_USE_URI=1
-    SQLITE_TEMP_STORE=1 # in memory
-    SQLITE_OMIT_LOAD_EXTENSION=1
-)
-
-target_include_directories(sqlite3 PUBLIC ${sqlite3_SOURCE_DIR})
+#include(FetchContent)
+#FetchContent_Declare(
+#  sqlite3
+#  URL      https://www.sqlite.org/2024/sqlite-amalgamation-3460100.zip
+#  URL_HASH SHA3_256=af6aae8d3eccc608857c63cf56efbadc70da48b5c719446b353ed88dded1e288
+#)
+#FetchContent_MakeAvailable(sqlite3)
+#add_library(sqlite3 STATIC ${sqlite3_SOURCE_DIR}/sqlite3.c)
+#target_compile_options(sqlite3 PRIVATE -w)  # disable warnings
+#
+#target_compile_definitions(sqlite3 PRIVATE
+#    SQLITE_ENABLE_API_ARMOR=1
+#    SQLITE_THREADSAFE=2
+#    SQLITE_ENABLE_FTS5=1
+#    SQLITE_ENABLE_FTS3_PARENTHESIS=1
+#    SQLITE_ENABLE_RTREE=1   # may need for loc-indexing
+#    SQLITE_ENABLE_COLUMN_METADATA=1
+#    SQLITE_USE_URI=1
+#    SQLITE_TEMP_STORE=1 # in memory
+#    SQLITE_OMIT_LOAD_EXTENSION=1
+#)
+#
+#target_include_directories(sqlite3 PUBLIC ${sqlite3_SOURCE_DIR})
 #############################################################################
 
 # link the LICENSE blob
